azurerm/internal/services/apimanagement/api_management_named_value_resource.go

```diff
@@ -46,11 +47,33 @@ func resourceApiManagementNamedValue() *pluginsdk.Resource {
 				ValidateFunc: validation.StringIsNotEmpty,
 			},
 
+			"key_vault": {
```

Comment:
i think this would be better UZ as 
```suggestion
			"value_from_key_vault": {
```

or

```suggestion
			"value_key_vault": {
```